pkg/xds/bootstrap/generator.go

```diff
@@ -153,6 +156,7 @@ func (b *bootstrapGenerator) generateFor(proxyId core_xds.ProxyId, dataplane *co
 	params := configParameters{
 		Id:                 proxyId.String(),
 		Service:            service,
+		Zone:               b.zone,
```

Comment:
is the zone in bootstrap still required?

Reply:
Removed.